ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-10.yaml

```
@@ -42,6 +42,16 @@ tests:
     - ref: stackrox-e2e
     workflow: ipi-aws
   timeout: 5h0m0s
+- as: ocp-sensor-integration-tests
```

openshift-ci will prefix the job with the variant name (ocp-4.10 in this case) so there is no need for `ocp-` in the `as:` values throughout. When your stackrox/stackrox PR merges they will show up as `ocp-4-*-sensor-integration-tests`.

```suggestion
- as: sensor-integration-tests
```

---

Done in [523a5f2](https://github.com/openshift/release/pull/40182/commits/523a5f2524aaf51ce7409d88883df750be38e7e0)